### ISSUE TITLE
Map should refresh list and/or Map should center on location that is clicked on

### ISSUE THREAD

**Issue by sumitsekhri** [findthemasks/findthemasks#620]
When a user clicks on a listing (we would need to set a target for a location (entire div or just location title) it should center the map around that location.

Secondly - it would be great if the list could refresh when the map is moved around (ex. Airbnb listings). 

Thanks!

**Comment by ngottlieb** [comment]
These are two pretty separate tasks, do you mind creating a second issue for the `List should sync with map based on map movement` issue?

**Comment by saikatbhadra** [comment]
Created https://github.com/findthemasks/findthemasks/issues/627 for the 2nd issue

@ngottlieb - thanks for keeping the issue queue clean!

**Comment by saikatbhadra** [comment]
I would put the task as aligning the map view port results with the list view as higher priority to this re-centering one.

**Comment by linderr** [comment]
I agree with you @saikatbhadra however this morning when you guys were going so crazy fast (amazing/impressive/heartening) @drwestco and @nburt noticed how bad the UX was about zoom issues on the map. 

They kind of already exist on the current live site but v2 just makes it even more apparent. Would definitely take @sumitsekhri's first suggestion more immediately now of allowing the whole div to be clickable real estate. When you load on a fresh map, the first instinct is to lick on the card to see where it is (default view is a whole country). It''s not intuitive in v2 to jump straight to the search bar as it was in v1

**Comment by ngottlieb** [comment]
Hmm, I've built map interfaces in the past that are similar -- a list and a map that sync with each other in real time as you use the map -- and my inclination is to say that a link with a zoom icon is more usable than having someone click the entire block. Without a CTA ("Click to zoom 🔍") people won't know that they can click on the blocks.

**Comment by linderr** [comment]
May we add that UI feature ? @sumitsekhri @tinabeans ?

**Comment by sumitsekhri** [comment]
@nburt that's a valid point. We can append the design to include this @linderr. 

I've also split the issue into two now.
